content/pt-br/docs/reference/setup-tools/kubeadm/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Kubeadm"
+weight: 10
+no_list: true
+content_type: concept
+card:
+  name: reference
+  weight: 40
+---
+
+<img src="/images/kubeadm-stacked-color.png" align="right" width="150px">O Kubeadm é uma ferramenta criada para fornecer o `kubeadm init` e o `kubeadm join` como "caminhos rápidos" de melhores práticas para criar clusters Kubernetes.
+
+O kubeadm executa as ações necessárias para colocar um cluster minimamente viável em funcionamento, e foi projetado para se preocupar apenas com a inicialização e não com o provisionamento de máquinas. Da mesma forma, a instalação de vários complementos úteis, como o Kubernetes Dashboard, soluções de monitoramento e complementos específicos da nuvem, não está no escopo.
+
+Em vez disso, esperamos que ferramentas de alto nível e mais personalizadas sejam construídas em cima do kubeadm e, idealmente, usando o kubeadm como base de todas as implantações torná mais fácil a criação de clusters em conformidade.
+
+## Como instalar
+
+Para instalar o kubeadm, consulte o [guia de instalação](/pt-br/docs/setup/production-environment/tools/kubeadm/install-kubeadm).
+
+## {{% heading "whatsnext" %}}
+
+* [kubeadm init](/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-init) para inicializar um nó da camada de gerenciamento do Kubernetes 
+* [kubeadm join](/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join) para inicializar um nó `worker` do Kubernetes e associá-lo ao cluster
+* [kubeadm upgrade](/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-upgrade) para atualizar um cluster Kubernetes para uma versão mais recente
```

Comment:
Among these links, the followings return 404 Page not found.

- https://kubernetes.io/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join
- https://kubernetes.io/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-upgrade

Reply:
Hi there, @jihoon-seo!

Thanks for pointing it out! We're holding this PR until all the linked pages get translated.